patches/components-sync-syncable-write_transaction.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/sync/syncable/write_transaction.h b/components/sync/syncable/write_transaction.h
+index 3d0a353ac09b1dcbfab0d5afc3345d78eb4ee8bd..eafebbf205de8691700162151ced152a4d5ba679 100644
+--- a/components/sync/syncable/write_transaction.h
++++ b/components/sync/syncable/write_transaction.h
+@@ -63,6 +63,7 @@ class WriteTransaction : public BaseTransaction {
+     transaction_ = trans;
+   }
+ 
++ BRAVE_SYNC_ATTACHABLE_WRITE_TRANSACTION_H
```

Comment:
same as below standard is BRAVE_WRITE_TRANSACTION_H_ and we should standardize the location. I think most are at the end of public. For the one below where you have it in private you can just do `private:` inside the define

Reply:
addressed in 3b1d457dc14d5a36353c45699459d728c5cb08ef